pkgs/mkpkgs

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env bash
+
+# NOTE: you'll need to create a arm64 buildx container first
+# `docker buildx create --platform linux/arm64 --use`
+# `docker buildx inspect <name of container> --bootstrap`
+# `docker buildx ls` should show it running & ready for arm64
+
+set -eux
+
+ARCH=${ARCH:-linux/arm64,linux/amd64}
+POSTGRES_VER=${POSTGRES_VER:-16}
+
+# FIND THE ROOT GIT DIRECTORY
+DIR=$(git rev-parse --show-toplevel $(dirname $0)|head -n 1)
+
+# ITERATION FOR THE PACKAGES
+ITERATION=${ITERATION:-1}
+
+# GIT REVISION OF SOURCE
+REVISION=${REVISION:-"$(git rev-parse --short HEAD)"}
+
+# READ VERSIONS
+source "${DIR}/versions.txt"
```

Comment:
Do we need this anywhere?

Reply:
It might have been leftover from a different tact. Let me check.

Reply:
deleted.